### PR TITLE
[READY] Server dev toasts

### PR DESCRIPTION
**Pull request by AffectedArc07**
## What Does This PR Do
Adds a config option to give you a toast notification when server inits are done
image

You can also enable it for GC ref finder
image

Inspired by https://github.com/tgstation/tgstation/pull/72465, but with some differences, most notably the toast creation is powered by a rust dependency inside of rustg instead of just a raw powershell script (note that rust still shows it as powershell, dont worry about it)

## Why It's Good For The Game
For those of us on weaker PCs, this can be useful to set your server starting in the background while you work on other things, then you can get a little notification when the server is ready for testing.

## Testing
I started the server, and saw a notification.

## Changelog
:cl:
add: [DEV] You can now get a notification when your local dev server has finished initialising 
add: [DEV] You can now get a notification when your local dev server has finished GC ref searching
/:cl:

### CODE REVIEW

File: code/__DEFINES/rust_g.dm [hunk 1]
```diff
@@ -268,6 +268,10 @@
 /proc/rustg_unix_timestamp()
 	return text2num(RUSTG_CALL(RUST_G, "unix_timestamp")())
 
+// Toast Operations //
+
+#define rustg_create_toast(title, body) RUSTG_CALL(RUST_G, "create_toast")(title, body)
```
Comment (AffectedArc07) [inline code/__DEFINES/rust_g.dm:273]: Dunno what else we would ever use this for, but its a method if we think of uses for it.

Maybe the GC ref finder?

**Comment by AffectedArc07** [comment]
Blocked by https://github.com/hoodie/notify-rust/issues/177

**Comment by AffectedArc07** [comment]
RUSTG Release Done https://github.com/ParadiseSS13/rust-g/releases/tag/v1.2.0-P2

File: code/__DEFINES/rust_g.dm (hunk 1 above)
Comment (AffectedArc07) [reply code/__DEFINES/rust_g.dm:273]: I'll implement that when I can later.

Comment (AffectedArc07) [reply code/__DEFINES/rust_g.dm:273]: Done in latest commit

**Review by S34NW — APPROVED** [review]
more external dependencies smh

File: code/_compile_options.dm [hunk 2]
```diff
@@ -22,6 +22,8 @@
 
 // #define FIND_REF_NO_CHECK_TICK	//Sets world.loop_checks to false and prevents find references from sleeping
 
+// #define FIND_REF_NOTIFY_ON_COMPLETE	// Throw a windows notificatioh toast when the ref finding process is done
```
Comment (SteelSlayer) [inline code/_compile_options.dm:25]:
```suggestion
// #define FIND_REF_NOTIFY_ON_COMPLETE	// Throw a windows notification toast when the ref finding process is done
```

**Review by SteelSlayer — APPROVED** [review]

**Pull request merged**